modules/csharp/examples/code_snippets/VectorSearch.cs

```diff
@@ -171,11 +170,12 @@ private void QueryUsingVectorMatch()
 
         private void QueryUsingVectorDistance()
         {
+            // tag::vs-use-vector-distance[]
```

Comment:
Ditto here, move the `var database` line out of the tag.  Also there is no real need to make the `inputColor` variable here, it could simply be used as a literal later.

Reply:
Done!